gcs_light_client.mediawiki

```diff
@@ -0,0 +1,992 @@
+<pre>
+BIP: ???
+Layer: Peer Services
+Title: Compact Client Side Filtering for Light Clients
+Author: Olaoluwa Osuntokun <laolu32@gmail.com>
+        Alex Akselrod <alex@akselrod.org>
+Comments: ???
+Comments-URI: ???
+Type: Standards Track
+Created: 05-24-2017
+License: PD
```

Comment:
Please choose one of the [recommended, or at least acceptable licenses](https://github.com/bitcoin/bips/blob/master/bip-0002.mediawiki#Recommended_licenses) (and also add a Copyright section below).

Reply:
Dunzo, we chose `CCO-1.0`.